modules/displayBooks.js

```diff
@@ -0,0 +1,26 @@
+export default function displayBooks() {
```

Comment:
- [ ] Please use the `arrow syntax` here. We are using ES6 Javascript in this milestone and every function should be an arrow function.